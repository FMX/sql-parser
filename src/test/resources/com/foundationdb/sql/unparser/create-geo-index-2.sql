create index if not exists idx1 on table1 (a, geo_lat_lon(c1, c2))
