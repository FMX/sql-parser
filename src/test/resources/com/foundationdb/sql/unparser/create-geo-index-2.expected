CREATE INDEX IF NOT EXISTS idx1 ON table1(a, geo_lat_lon(c1, c2))